pp/integrations/plugin-packs/procedures/applications-pacemaker-ssh.md

```diff
@@ -2,40 +2,431 @@
+| WARNINGRESOURCEACTIONSFAILED         | Thresholds                                                                                                                                                                                                                   |                                    |             |
+| CRITICALRESOURCEACTIONSFAILED        | Thresholds                                                                                                                                                                                                                   |                                    |             |
+| CRITICALRESOURCEMIGRATIONFAILED      | Thresholds                                                                                                                                                                                                                   | 0                                  |             |
+| WARNINGRESOURCEMIGRATIONFAILED       | Thresholds                                                                                                                                                                                                                   |                                    |             |
+| CRITICALRESOURCESTATUS               | Define the conditions to match for the status to be CRITICAL (default: '%{status} =~ /stopped\|failed/i'). You can use the following variables: %{name}, %{status}, %{node}, %{is\_unmanaged}                                | %{status} =~ /stopped\|failed/i    |             |
+| WARNINGRESOURCESTATUS                | Define the conditions to match for the status to be WARNING. You can use the following variables: %{name}, %{status}, %{node}, %{is\_unmanaged}                                                                              |                                    |             |
+| EXTRAOPTIONS                         | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                                                                                           | --verbose                          |             |
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+</TabItem>
+<TabItem value="Clustat" label="Clustat">
 
-| Field                   | Value                      |
-| :---------------------- | :------------------------- |
-| Host name               | *Name of the host*         |
-| Alias                   | *Host description*         |
-| IP                      | *Host IP Address*          |
-| Monitored from          | *Monitoring Poller to use* |
-| Host Multiple Templates | App-Pacemaker-SSH-custom   |
+| Macro        | Description                                                                                        | Default value     | Mandatory   |
```

Comment:
```suggestion
> This service template is deprecated and is no longer provided with the latest versions of the connector since the `clustat` is no longer part of Pacemaker's commande-line tools since RHEL7.

| Macro        | Description                                                                                        | Default value     | Mandatory   |
```